DataFormats/Luminosity/src/classes_def.xml

```diff
@@ -14,8 +14,8 @@
   <class name="LumiInfoRunHeader"  ClassVersion="10">
    <version ClassVersion="10" checksum="3459250671"/>
   </class>
-  <class name="LumiInfo"  ClassVersion="10">
-   <version ClassVersion="10" checksum="4234044727"/>
```

Comment:
Please, still keep track of previous ClassVersion for backward compatibility: this cancelled out line should be put back below current L18 (see all other classes in this file)